src/platform-includes/performance/automatic-performance-metrics/android.mdx

```diff
@@ -2,3 +2,15 @@ If configured, the Android SDK automatically collects the following performance
 
 * <PlatformLink to="/performance/instrumentation/automatic-instrumentation/#app-start-instrumentation">Cold and warm app start time</PlatformLink>.
 * <PlatformLink to="/performance/instrumentation/automatic-instrumentation/#slow-and-frozen-frames">Slow and frozen frame rendering</PlatformLink>.
+
+### Application Not Responding (ANR) Rate
+
+<Include name="early-adopter-note.mdx" />
+
+If configured, the Android SDK reports [Application Not Responding (ANR)](platforms/android/configuration/app-not-respond/) errors if the main UI thread of the application is blocked for more than five seconds. ANR rate is the percentage of distinct users who have experienced any kind of ANR during the specified time period. Foreground ANR rate is the percentage of distinct users who have experienced an ANR when the app was running in the foreground. It is more likely that foreground ANRs were perceieved by the user.
+
+ANR rate is displayed on one of the cards in the Project Details page for Android projects. The 'View Issues' button on the score card takes you to a list of ANR issues that were created in the selected timeframe and environment. The Project Details page also shows an ANR rate graph by default. Toggle the **Display** to view foreground ANR rate.
+
+User-perceieved ANR rate is defined as the unique percentage of users that have experienced the app not responding to an input event (such as key press or screen touch) within 5 seconds. It is a core Android vital that has an [overall bad behaviour threshold](https://developer.android.com/topic/performance/vitals#what_are_the_bad_behavior_thresholds) of 0.47% set by Google Play. If your app exceeds the overall bad behaviour threshold, Google Play may reduce the visibility of your app. Foreground ANR rates may not map exactly to user-perceieved ANR rates but could provide an effective proxy. The foreground ANR rate chart on the Project Details page also plots the bad behaviour threshold line for reference.
+
+ANR rate and Foreground ANR rate is supported in Sentry's Android SDK version x.x.x and above.
```

Comment:
```suggestion
### User-perceived ANR Rate

User-perceived ANR rate is a core Android vital and is defined more specifically, as the unique percentage of users who've experienced the lack of response to an input event in an app, lasting more than five seconds. 

A user-perceived ANR rate that's more than 0.47%, is considered over the [bad behavior threshold](https://developer.android.com/topic/performance/vitals#what_are_the_bad_behavior_thresholds), as set by Google. If your app exceeds this threshold, Google Play may choose to reduce your app’s visibility in their app store.

### Foreground ANR Rate

In Sentry, we can monitor the foreground ANR rate, which is the percentage of unique users who’ve experienced an ANR event lasting more than five seconds, while the app was running in the foreground. Because it’s more noticeable to the user, a foreground ANR event is more likely to be seen as an annoyance.

While foreground ANR rates may not map exactly to user-perceived ANR rates, they can provide a close proxy. In Sentry, the foreground ANR rate chart has a bad behavior threshold line for comparison.

To see your ANR rate in Sentry, go to the **Projects** page, click on your Android project, then look for the ANR rate card in the "Project Details" section. Clicking the "View Issues" button in the "ANR Rate" card will take you to a list of ANR issues that were created in the selected timeframe and environment. 

You can also see an ANR rate graph on a separate card. Opening the "Display" menu will allow you to switch back and forth between the "ANR Rate" and the "Foreground ANR Rate" view.

```